INSTALL

```diff
@@ -92,7 +92,7 @@
 
   on Windows (only pick one of the targets for configuration):
 
-    $ perl Configure { VC-WIN32 | VC-WIN64A | VC-WIN64I | VC-CE }
+    $ perl Configure { VC-WIN32 | VC-WIN32-core  | VC-WIN64A | VC-WIN64I | VC-WIN64A-core | VC-CE }
```

Comment:
Change `core` to `ONECORE`

Reply:
actually i just was thinking about totally removing it, wonder if it is necessary here as it's additional conf specified in separate conf file, there is lot of them and they are not mentioned here

Reply:
There's another difference...  the others are treated with a different script that figures out the correct config target.  With those, you don't *have* to tell the configuration what target you intend to build on.  With native Windows, it doesn't work the same way, you have to be explicit.  You'll notice that those we already have there are all of those we support, so the question is if we want ONECORE to be a supported variant or not...  time will tell, I guess ;-)